src/NServiceBus.Transport.SQS/Configure/SqsTransport.cs

```diff
@@ -57,6 +57,11 @@ internal set
         /// </summary>
         public string QueueNamePrefix { get; set; }
 
+        /// <summary>
+        /// Disable native delayed delivery infrastructure
+        /// </summary>
+        internal bool DisableDelayedDelivery { get; set; } = false;
```

Comment:
I understand this property name has been used in various places. I'm not recommending a change at this point, but I'm pointing out that using double negatives can sometimes be confusing to readers. If EnableDelayedDelivery were set to true instead, it might be easier to comprehend quickly. This is something to consider for future cases when instincts might lead you to code double negative defaults.